src_c/draw.c

```diff
@@ -710,8 +710,8 @@ circle(PyObject *self, PyObject *args, PyObject *kwargs)
 
     CHECK_LOAD_COLOR(colorobj)
 
-    if (radius < 0) {
-        return RAISE(PyExc_ValueError, "negative radius");
+    if (radius < 1) {
+        return pgRect_New4(posx, posy, 0, 0);
```

Comment:
You could combine this check with the width check on line 717 since they are now returning the same thing.

Reply:
Good idea. Thanks.